src/Microsoft.AspNetCore.Razor.Design/Internal/AssemblyTagHelperDescriptorResolver.cs

```diff
@@ -61,6 +52,47 @@ public IEnumerable<TagHelperDescriptor> Resolve(string assemblyName, ErrorSink e
                         DesignResources.InvalidProtocolValue,
                         typeof(TagHelperDescriptor).FullName, ProtocolVersion));
             }
+
+            if (!_loadedAssembly)
```

Comment:
In which case, could we create a delegate from `populateMethod` (https://docs.microsoft.com/en-us/dotnet/core/api/system.reflection.methodinfo#System_Reflection_MethodInfo_CreateDelegate_System_Type_)